src/modules/apps/link.ts

```diff
@@ -212,6 +224,13 @@ export default async (options) => {
   if (options.setup || options.s) {
     await setup()
   }
+  try {
+    const aux = await getPinnedDependencies(builderHttp)
+    const pinnedDeps : Map<string, string> = new Map(Object.entries(aux))
+    await bluebird.map(buildersToRunLocalYarn, fixPinnedDependencies(pinnedDeps), { concurrency: 1 })
+  } catch(e) {
+    log.info('Failed to check for pinned dependencies')
```

Comment:
I think this message should be changed to something like `Failed setting pinned dependencies`